docs/resources/control-flow/user-interactivity/forms/form-actions/reset-form-field.md

```diff
@@ -0,0 +1,31 @@
+---
+slug: reset-form-field
```

Comment:
only add custom slugs for the forms section (need to be shortened because original is too big)

this would be like
slug: resources/forms/reset-form-field

Reply:
Done!